test/integration/narrow/routes/authenticate.test.js

```diff
@@ -1,3 +1,5 @@
+jest.mock('ffc-messaging')
```

Comment:
There were quite a few of these missing which meant that a new connection was created for every test run.  Added some where I could see the issue.